alpha/lib/model/assetPeer.php

```diff
@@ -851,4 +851,37 @@ public static function retrieveAudioFlavorsByEntryIdAndDimensions($entryId, arra
 		$c->add(assetPeer::TYPE, assetType::FLAVOR);
 		return assetPeer::doSelect($c);
 	}
+	/**
+	 *
+	 * @param string $entryId
+	 * @param string $tag
+	 * @return array<assets>
+	 */
+	public static function retrieveByEntryIdAndTag($entryId, $tag)
+	{
+		$entryThumbAssets = self::retrieveThumbnailsByEntryId($entryId);
+		return self::filterByTag($entryThumbAssets, $tag);
+	}
+
+
+	/**
+	 * Returns true if this is the only default thumb and there are temp thumbs.
+	 *
+	 * @param string $thumbAssetId
+	 * @param string $entryId
+	 * @return bool
+	 */
+	public static function shouldSetAsDefaultThumb($thumbAssetId,$entryId)
```

Comment:
space between params
also i think the name is confusing and doesn't represent the actual logic, better rename to something like shouldReplaceTemporaryDefaultThumb